src/components/ScanVC.tsx

```
@@ -59,14 +59,15 @@ const ScanVC: React.FC<ScanVCProps> = ({ onScanResult }) => {
 					if (!hasScanned.current) {
 						hasScanned.current = true;
+						//NOSONAR_BEGIN
 						/* toast({
 							title: 'Scan Success',
 							description: decodedText,
 							status: 'success',
 							duration: 2000,
 							isClosable: true,
 						});
- 						*/	// NOSONAR
+ 						*/	//NOSONAR_END
```

ESLint

[error] Replace `↹` with `·`

(prettier/prettier)